developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -343,12 +355,49 @@ You can request properties of a folder without also getting the folder contents
 Downloading files
 -----------------
 
+.. note:: The :code:`GET` method is not defined by the WebDAV standard, this is a Nextcloud provided WebDAV extension.
```

Comment:
```suggestion
```
copy-pasta

Reply:
No that is intended, `GET` is not defined by WebDAV neither for files nor folders. So I added this to folders to make clear the zip function is Nextcloud only. But the same is valid for files.

I can remove if you like but would keep this for consistency otherwise.